### ISSUE TITLE
Bug: Risky feature tests with Controller returns

### ISSUE THREAD

**Issue by MGatner** [codeigniter4/CodeIgniter4#3063]
**Describe the bug**
`FeatureTestCase` allows simulating an app run and gathering the output for testing purposes. If a controller returns the output instead of echoing it (e.g. `return view('Home');`) it isn't cleaning the output buffer correctly causing risky tests with the following warning:

> Test code or tested code did not (only) close its own output buffers

**CodeIgniter 4 version**
`develop` latest

**Affected module(s)**
`FeatureTestTrait`

**Expected behavior, and steps to reproduce if appropriate**
Controller:
```
class Home extends BaseController
{
	public function index()
	{
		return view('home');
	}
```

Test:
```
class HomeTest extends FeatureTestCase
{
	public function testGetHome()
	{
		$result = $this->get('/');

		$result->assertStatus(200);
```

**Context**
 - OS: Linux
 - Web server: Apache
 - PHP version: 7.3

**Comment by MGatner** [comment]
@samsonasik any chance this is an easy fix similar to #2551 or #2625? I honestly have dug into it yet but won't have a chance to today.

**Comment by samsonasik** [comment]
I tried do buffering check in the FeatureTestCase class if i remember correctly, but not work. I hope I can re-check again if I have a chance.

**Comment by samsonasik** [comment]
@MGatner should can be closed in favor of #3083 ?

**Comment by MGatner** [comment]
Yes! Thanks for catching that. I had a “fixes ____” commit but I must have overwritten it when I force pushed.
